app/models/purchase.py

```diff
@@ -2,33 +2,77 @@
 from datetime import datetime
 import pytz
 
+
+class CartItem(db.Model):
+    ''' Functions as association table between listings and merchants '''
+    __tablename__ = "cartItems"
+    merchant_id = db.Column(db.Integer, db.ForeignKey('users.id'),
+                            primary_key=True)
+    listing_id = db.Column(db.Integer, db.ForeignKey('listings.id'),
+                           primary_key=True)
+    quantity = db.Column(db.Integer)
+
+    listing = db.relationship("Listing")
+
+
+class Status:
+    PENDING = 0
+    APPROVED = 1
+    DECLINED = 2
+
+
+class Order(db.Model):
+    __tablename__ = 'orders'
+    id = db.Column(db.Integer, primary_key=True)
+
+    date = db.Column(db.DateTime)
+    status = db.Column(db.Integer)
+
+    def __init__(self, cart_items):
```

Comment:
So, this is strange and maybe clever. Essentially, this constructor lets you initialize an `Order` with a `Merchant`'s `cart_item` and the Order will automatically populate its `purchases` with newly constructed `Purchase` objects.